lib/loggers/index.js

```diff
@@ -16,7 +19,8 @@ const createLogger = opts => {
   const { getLogger, getLogLevels } = require(`./${loggerOption}`);
   const logger = getLogger(options);
   const loggerWrapper = getLogLevels(logger).reduce((newLogger, logLevel) => {
-    newLogger[logLevel] = (...args) => logger[logLevel](getRequestIdPrefix(), ...args);
+    newLogger[logLevel] =
+      isTesting && !logOnTesting ? () => true : (...args) => logger[logLevel](getRequestIdPrefix(), ...args);
```

Comment:
Maybe could be implemented as a key in `opts`, also add documentation in readme

Reply:
@mpicollo-wolox I've changed it to be an opts flag. Now, it depends on the implementations to know if it should log or not.
Also, I've added a test to validate the new option.